### DISCUSSION TITLE
tokio::prelude::future::join_all

### DISCUSSION THREAD

**Discussion by vladgon** [tokio-rs/tokio#2574] [General] [answered]
https://docs.rs/tokio/0.1.22/tokio/prelude/future/fn.join_all.html

tokio::prelude::future::join_all was  removed in tokio 0.2.21, there is an option to use 
futures::future::try_join_all
futures::future::join_all

Please advice 

Thanks

**Comment by Darksonn** [comment, marked as answer]
You should use [`join_all`][1] from the futures crate for joining on all items in a vector now.

[1]: https://docs.rs/futures/0.3/futures/future/fn.join_all.html
